From ef98334b4b1586a2bcb0cd9d48b2d12299be37ad Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
Date: Fri, 19 Apr 2024 12:33:52 +0300
Subject: [PATCH] Do not run test programs when cross compiling

---
 configure.ac | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/configure.ac b/configure.ac
index d42fbea..21a855b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -44,14 +44,6 @@ AC_CHECK_HEADERS([float.h limits.h stddef.h stdlib.h string.h sys/time.h stdint.
 AS_BOX([       START GIVARO CONFIG       ])
 ###########################################
 
-AC_COMPILER_NAME
-
-# We need a C++11 compiler now - AB 2014-12-12
-# clang-3.8 does not support __float128 without explicitly passing it -std=c++11
-AS_IF([test "x${CCNAM}" = "xclang38"],
-      [AX_CXX_COMPILE_STDCXX_11([noext],[mandatory])],
-      [AX_CXX_COMPILE_STDCXX_11([ext],[mandatory])]
-      )
 REQUIRED_FLAGS="${CXX11FLAGS}"
 # Add the c++11 flags for the configure compilations as clang needs it to work properly with a glibc++ compiled with gcc
 CXXFLAGS="${CXX11FLAGS} ${CXXFLAGS}"
@@ -60,9 +52,6 @@ AS_ECHO([---------------------------------------])
 
 AC_INLINE # DON'T TOUCH
 
-# Set OPTIM_FLAGS, DEBUG_FLAGS depending on compiler and command line arguments
-SET_FLAGS
-
 # Append -march=native or -mcpu=native (if recognized by the compiler) to
 # OPTIM_FLAGS if not present in CXXFLAGS and not cross-compiling and
 # --without-archnative is not set
@@ -120,8 +109,8 @@ done
 AS_ECHO([---------------------------------------])
 
 # Look for GMP and add flags if necessary
-GIV_CHECK_GMP(40000)
 REQUIRED_FLAGS+=" ${GMP_CFLAGS}"
+GMP_LIBS="-lgmp"
 
 GIV_DOC
 
-- 
2.44.0

