--- configure.ac        2021-03-20 18:21:44.105930027 +0000
+++ configure.ac        2021-03-20 18:22:15.162257892 +0000
@@ -45,8 +45,6 @@
 #set CXX
 AC_PROG_CXX
 
-AC_COMPILER_NAME
-
 # We need a C++11 compiler now - AB 2014-12-12
 # clang-3.8 does not support __float128 without explicitly passing it -std=c++11
 if test "x${CCNAM}" = "xclang38" ; then
@@ -123,7 +121,6 @@
 echo "-----------------------------------------------"
 
 # Checks which SIMD optimization flags to use and set SIMD_CFLAGS accordingly
-INSTR_SET
 AC_SUBST(SIMD_FLAGS)
 
 # Veltkamp Split in modular-extended.inl requires to use the SSE floating point multiplier and not the fp87

diff --git a/macros/gmp-check.m4 b/macros/gmp-check.m4
index 3058cc2..77ea9ac 100644
--- macros/gmp-check.m4
+++ macros/gmp-check.m4
@@ -115,7 +115,9 @@ AC_DEFUN([GIV_CHECK_GMP], [
 		],
 		[ AC_MSG_RESULT(no)
 		  AC_MSG_ERROR(your GMP is too old. GMP release >= $min_gmp_release needed)
-		  exit 1]
+		  exit 1],
+        [ AC_MSG_RESULT(yes)
+        ]
 	)
 	AC_LANG_POP([C++])
 	
